### PR TITLE
Updating to `v0.4.3`

### PR DESCRIPTION
**Pull request by sugatoray**
Additionally:
- adding description, doc_url and dev_url
- adding `sugatoray` as co-maintainer

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sugatoray** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,8 +40,29 @@ about:
...
+    and assessment user interfaces and libraries that enable a better understanding of AI systems.
+    These interfaces and libraries empower developers and stakeholders of AI systems to develop and
+    monitor AI more responsibly, and take better data-driven actions.
+
+    - `{{ name | lower }}` is one of the tools of RAI Toolbox.
+
+    <p align="center">
+    <img src="https://raw.githubusercontent.com/microsoft/responsible-ai-widgets/main/img/responsible-ai-toolbox.png" alt="ResponsibleAIToolboxOverview" width="750"/>
+
+    PyPI: [https://pypi.org/project/{{ name | lower }}/](https://pypi.org/project/{{ name | lower }}/)
```
Comment (janjagusch) [inline recipe/meta.yaml]: I'm unsure how the image would render (and where).

```suggestion
  description: |
    **Responsible AI Toolbox**: `{{ name | lower }}`

    Responsible AI is an approach to assessing, developing, and deploying AI systems in a safe,
    trustworthy, and ethical manner, and take responsible decisions and actions.

    Responsible AI Toolbox is a suite of tools providing a collection of model and data exploration
    and assessment user interfaces and libraries that enable a better understanding of AI systems.
    These interfaces and libraries empower developers and stakeholders of AI systems to develop and
    monitor AI more responsibly, and take better data-driven actions.

    - `{{ name | lower }}` is one of the tools of RAI Toolbox.

    PyPI: [https://pypi.org/project/{{ name | lower }}/](https://pypi.org/project/{{ name | lower }}/)
```

**Review by janjagusch — COMMENTED** [review]
thanks!

File: recipe/meta.yaml (hunk 1 above)
Comment (sugatoray) [reply recipe/meta.yaml]: I have previously added images. It works. However, I am fine with your suggestion as well. Thank you.

cc: @janjagusch

**Review by bgalvao — APPROVED** [review]
LGTM

**Pull request merged**